seissolxdmfwriter/seissolxdmfwriter/seissolxdmfwriter.py

```diff
@@ -145,22 +210,61 @@ def write_seissol_output(
     lidt: list of time steps to be written
     reduce_precision: convert double to float and i64 to i32 if True
     """
+    print("Warning: write_seissol_output is deprecated. Please use write instead")
```

Comment:
You can even warn here, cf. https://stackoverflow.com/a/9008509

Reply:
done